bundle/manifests/metallb-operator.clusterserviceversion.yaml

```diff
@@ -430,6 +436,90 @@ spec:
                 - list
                 - watch
           serviceAccountName: controller
+        - rules:
```

Comment:
the rule for metallb to consume the frr configuration are missing, but I guess they'll come when the integration lands, right?